guides/doc-Release_Notes/master.adoc

```diff
@@ -14,6 +14,7 @@ endif::[]
 
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.8.0.adoc[leveloffset=+1]
+include::topics/katello-4.10.0.adoc[leveloffset=+1]
```

Comment:
This is only for the Katello build. We also typically list Katello before Foreman. Mostly because the releases are chronological (from newest to oldest) and Katello is only released GA after Foreman.
```suggestion
ifdef::katello[]
include::topics/katello-4.10.0.adoc[leveloffset=+1]
endif::[]
include::topics/foreman-3.8.0.adoc[leveloffset=+1]
```